testgrid/config.yaml

```diff
@@ -2507,60 +2507,72 @@ test_groups:
   gcs_prefix: origin-federated-results/pr-logs/directory/test_pull_request_crio_e2e_fedora
 - name: test_pull_request_crio_e2e_rhel
   gcs_prefix: origin-federated-results/pr-logs/directory/test_pull_request_crio_e2e_rhel
-# Istio
-- name: istio-istio-postsubmit
-  gcs_prefix: istio-prow/istio-postsubmit
-  num_failures_to_alert: 1
-- name: istio-e2e-suite-rbac-no_auth
-  gcs_prefix: istio-prow/e2e-suite-rbac-no_auth
-  num_failures_to_alert: 1
-- name: istio-e2e-suite-rbac-auth
-  gcs_prefix: istio-prow/e2e-suite-rbac-auth
-  num_failures_to_alert: 1
-- name: istio-e2e-cluster_wide-auth
-  gcs_prefix: istio-prow/e2e-cluster_wide-auth
-  num_failures_to_alert: 1
 # Istio Prow Presubmits
 - name: istio-presubmit
   gcs_prefix: istio-prow/directory/istio-presubmit
-- name: istio-unit-tests
+- name: istio-unit-tests-presubmit
   gcs_prefix: istio-prow/directory/istio-unit-tests
-- name: istio-pilot-e2e
+- name: istio-pilot-e2e-presubmit
   gcs_prefix: istio-prow/directory/istio-pilot-e2e
-- name: istio-pilot-e2e-envoyv2-v1alpha3
+- name: istio-pilot-e2e-envoyv2-v1alpha3-presubmit
   gcs_prefix: istio-prow/directory/istio-pilot-e2e-envoyv2-v1alpha3
-- name: e2e-simpleTests
+- name: e2e-mixer-no_auth-presubmit
```

Comment:
Done

Reply:
It is important to have the `istio-` prefix in the test_group because these are globally unique.

On the other hand, I would recommend stripping anything redundant from your dashboard names, since these are local to the dashboard.

In particular, having every tab in the istio dashboard start with `istio-` is a waste of space -- and the reason tabs have both a `name` and `test_group` field is to allow the locally-scoped name to differ from the globally-scoped group.

Reply:
Thanks for the tips! I have removed the redundant prefix in dashboard names as suggested.